test/util/handleChange.spec.js

```diff
@@ -0,0 +1,17 @@
+import handleChange from '../../src/util/handleChange';
+
+describe('HandleChangeText', () => {
+    it('Should change answer to a happy answer', () => {
```

Comment:
los `it` y `context` tienen que empezar con minus porque es la continuación de la oración.